src/CIFAR100/CIFAR100.jl

```diff
@@ -0,0 +1,87 @@
+export CIFAR100
+module CIFAR100
+    using DataDeps
+    using BinDeps
+    using FixedPointNumbers
+    using ..bytes_to_type
+    using ..datafile
+    using ..download_dep
+    using ..download_docstring
+    import ..CIFAR10.convert2image
+    import ..CIFAR10.convert2features
+
+    export
+
+        classnames_coarse,
+        classnames_fine,
+
+        traintensor,
+        testtensor,
+
+        trainlabels,
+        testlabels,
+
+        traindata,
+        testdata,
+
+        convert2image,
+        convert2features,
+
+        download
+
+    const DEPNAME = "CIFAR100"
```

Comment:
Is this good flexible code?
Or is it actually more confusing than it is worth?
I'm not sure.

I guess it lets you copy and past lines that mean the same thing that use the same variable names, in different submodules

Reply:
Well this defines the name of the folder/registers datadep. Given that it is basically the name of the dataset it made sense to me to make it into a variable. Otherwise, changing that at any time would be a pain, since you can't easily search and replace it.